lambdas/evaluate-gpg45-scores/src/main/java/uk/gov/di/ipv/core/evaluategpg45scores/EvaluateGpg45ScoresHandler.java

```diff
@@ -178,6 +178,8 @@ public APIGatewayProxyResponseEvent handleRequest(
 
             updateSuccessfulVcStatuses(ipvSessionItem, credentials);
 
+            checkCorrelation(userId, ipvSessionItem.getCurrentVcStatuses());
```

Comment:
Why don't we check the correlation against the credentials we already have since it looks like we have to parse the JWTs further down anyway?